test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -384,20 +385,26 @@ PROTO_TOSTRING(LastStatement, laststat)
 	using LastStatType = LastStatement::LastOneofCase;
 	switch (laststat.last_oneof_case()) {
 	case LastStatType::kExplist:
-		laststat_str = ReturnOptionalExpressionListToString(
-			laststat.explist());
+		if (GetContext().return_is_possible()) {
```

Comment:
We may add this condition as the part of `ReturnOptionalExpressionListToString()` instead. Here and below.
Feel free to ignore.

Reply:
Moved to `ReturnOptionalExpressionListToString()`.